sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_create_test_live.cpp

```diff
@@ -1,16 +1,28 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-#include "gtest/gtest.h"
-
+#include "../../azure-security-attestation/src/private/crypto/inc/crypto.hpp"
```

Comment:
Why is this needed? At a glance, I don't see anything in that header we need in this file.

Each SDK should be self-contained and individually buildable/testable, other than a dependency on core (and identity). Attestation and Keyvault are fairly independent.

Reply:
done